### ISSUE TITLE
TensorFlow model optimisation

### ISSUE THREAD

**Issue by namannimmo10** [DiamondLightSource/ml4deltabot#5]
Useful links:

https://www.tensorflow.org/model_optimization/guide

**Quantisation**

Post training quantisation:
https://ai.google.dev/edge/litert/conversion/tensorflow/quantization/post_training_quantization

Quantisation aware training:
https://www.tensorflow.org/model_optimization/guide/quantization/training
https://www.tensorflow.org/model_optimization/guide/quantization/training_example

Compare the loss of accuracy

**Pruning**

https://www.tensorflow.org/model_optimization/guide/pruning/pruning_with_keras
